doc/Logging.md

```diff
@@ -0,0 +1,16 @@
+How does Happy test log work
```

Comment:
Let's make the filename lowercase logging.md

I also have a general documentation style guide under review that you might want to reference, let's start keeping everything the same style as much as possible: https://github.com/openweave/openweave-core/blob/84721a01b2a4c0b1476e0059b7f185b19c4c7955/doc/STYLE_GUIDE.md

And use a title header and remove line 2

# How Happy test logs work

Reply:
done